scss/components/_dropdown-menu.scss

```diff
@@ -152,7 +152,7 @@ $dropdown-menu-item-background-active: transparent !default;
     }
 
     // Active state
-    .is-active > a {
+    .is-active {
       background: $dropdown-menu-item-background-active;
```

Comment:
This breaks existing projects that rely on `is-active` to be on the links's parent instead of on the link itself.
Thus I wouldn't change this or extend it:
```sass
.is-active {
    &, > a {
```